tests/recipes/trig.rs

```diff
@@ -33,6 +33,7 @@ pub fn trig_rules() -> Ruleset<Trig> {
         .clone()
         .plug("op", &t_ops)
         .plug("v", &consts)
+        .append(Workload::new(["0", "1"]))
```

Comment:
I believe some of the other workloads added `[-2, -1, 0, 1, 2]`. You could add those to be consistent?